Prace_domowe/Praca_domowa2/Jakub_Foltyn/PD2.ipynb

```diff
@@ -0,0 +1,1178 @@
+{
```

Comment:
<p>należałoby zaprezentować jak wygląda ramka danych po zastosowaniu encodingu, bo to właśnie w takiej postaci dane będą używane na etapie modelowania. Dotyczy to także wariantów prezentowanych poniżej.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>